spec/mailers/rrd_completed_mailer_spec.rb

```diff
@@ -28,7 +28,10 @@
                                                               form_json: form_json)
       # Set the bp_readings_count like `add_medical_stats` is expected to do
       form_json = JSON.parse(submission.form_json)
-      form_json['rrd_metadata'] = { med_stats: { bp_readings_count: bp_readings_count } }
+      form_json['rrd_metadata'] = {
+        med_stats: { bp_readings_count: bp_readings_count },
+        pdf_guid: 'a950ef07-9eaa-4784-b5af-bda8c50a83f9'
+      }
```

Comment:
Can we verify this is the real structure? Even just spot-check since we're mocking it.

Reply:
Yes, good question. This test ensures it's there: https://github.com/department-of-veterans-affairs/vets-api/blob/90e350090d5afb98604e370a6d0cc732c6ab2e91/spec/services/rapid_ready_for_decision/fast_track_pdf_upload_manager_spec.rb#L101-L108

If you're referring to an higher-level test, I didn't think this was important enough to warrant that.

Reply:
Good enough for me! I agree.